### ISSUE TITLE
cli-tools fail with no action specified in python3

### ISSUE THREAD

**Issue by cladmi** [iot-lab/iot-lab#173]
When executing cli-tools without any commands, the program raises a KeyError instead of an Argparse error.

Python3:

```
 python3.4 experiment-cli 
Traceback (most recent call last):
  File "experiment-cli", line 4, in <module>
    iotlabcli.parser.experiment.main()
  File "/home/harter/work/iot-lab/parts/cli-tools/iotlabcli/parser/experiment.py", line 441, in main
    common.main_cli(experiment_parse_and_run, parser, args)
  File "/home/harter/work/iot-lab/parts/cli-tools/iotlabcli/parser/common.py", line 49, in main_cli
    result = function(parser_opts)
  File "/home/harter/work/iot-lab/parts/cli-tools/iotlabcli/parser/experiment.py", line 432, in experiment_parse_and_run
    }[opts.command]
KeyError: None
```

Expected behaviour
```
 python2.7 experiment-cli 
usage: experiment-cli [-h] [-u USERNAME] [-p PASSWORD] [-v]
                      {submit,stop,get,load,info,wait} ...
experiment-cli: error: too few arguments
```

**Comment by cladmi** [comment]
Related:

http://bugs.python.org/issue9253
http://stackoverflow.com/questions/22990977/why-does-this-argparse-code-behave-differently-between-python-2-and-3
http://stackoverflow.com/questions/23349349/argparse-with-required-subparser

**Comment by cladmi** [comment]
Fixed in version 1.5.0:

https://github.com/iot-lab/cli-tools/commit/b3655f3397cf271da36d59bce2dce374f8a1906d
